include/xtensor/xchunk_store_manager.hpp

```diff
@@ -260,6 +263,13 @@ namespace xt
         m_shape = shape;
     }
 
+    template <class EC, class IP>
+    template <class S>
+    inline void xchunk_store_manager<EC, IP>::set_chunk_shape(S&& chunk_shape)
+    {
+        m_chunk_pool[0].resize(chunk_shape);
```

Comment:
What if the chunk pool contains more than one chunk?

Reply:
For that, `set_pool_size()` has to be called, and [the shape is set on every chunk of the pool](https://github.com/xtensor-stack/xtensor/blob/3375147a9fc01d20aa6572c0e6b0dcd791cf7c7a/include/xtensor/xchunk_store_manager.hpp#L284-L293).
Also [`set_pool_size()` is called after `set_chunk_shape()`](https://github.com/xtensor-stack/xtensor/blob/3375147a9fc01d20aa6572c0e6b0dcd791cf7c7a/include/xtensor/xchunked_array.hpp#L271).

Reply:
I meant if `set_chunk_shape` is called after `set_pool_size` has been called.